pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -189,12 +190,13 @@ def write_pep0(self, peps: list[PEP], header: str = HEADER, intro: str = INTRO,
             if status in HIDE_STATUS:
                 continue
```

Comment:
```suggestion
            status_code = "<none>" if status == STATUS_DRAFT else status[0]
```

This whole block of if statements can be replaced with just this line, as Accepted/Active no longer need to be special cased (as their descriptions explain the difference better than the old bespoke text), and Draft PEPs should still be mentioned here somehow, since it doesn't state anywhere in the document that PEPs with an empty status code are draft PEPs.

Also, `HIDE_STATUS` in `pep_zero_generator.constants` can consequently be removed.